engine/flink/api/src/main/scala/pl/touk/nussknacker/engine/flink/api/process/RichLifecycleFunction.scala

```diff
@@ -0,0 +1,30 @@
+package pl.touk.nussknacker.engine.flink.api.process
+
+import org.apache.flink.api.common.functions.{AbstractRichFunction, MapFunction, RuntimeContext}
+import org.apache.flink.configuration.Configuration
+import pl.touk.nussknacker.engine.api.Lifecycle
+import pl.touk.nussknacker.engine.api.runtimecontext.EngineRuntimeContext
+
+abstract class RichLifecycleFunction extends AbstractRichFunction {
```

Comment:
Do we use it ATM?

Reply:
no, I can add it in next change when it will be needed (wanted to have in those change only functional changes)